tools/java-source-utils/java-source-utils.csproj

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This should probably use `<Project Sdk="Microsoft.Build.NoTargets/1.0.110">` which is what we use for things that aren't actually invoking `csc` or creating a managed assembly: https://github.com/xamarin/java.interop/blob/master/src/java-interop/java-interop.csproj.

Reply:
Done in latest force-push.